src/main/java/racingcar/service/RacingService.java

```diff
@@ -0,0 +1,79 @@
+package racingcar.service;
+
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.domain.CarStateInRace;
+import racingcar.domain.MovingResult;
+import racingcar.domain.RacingResult;
+import racingcar.domain.movable.RandomForwardStrategy;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class RacingService {
+
+    private final Cars cars;
+    private final MovingResult movingResult;
+    private final RacingResult racingResult;
+
+    private RacingService(final Cars cars) {
+        this.cars = cars;
+        this.movingResult = MovingResult.instance();
+        this.racingResult = RacingResult.instance();
+    }
```

Comment:
```suggestion
    private RacingService(final Cars cars) {
        this(cars, MovingResult.instance(), RacingResult.instance())
    }
```
메서드 체이닝을 이용해보는 것은 어떨까요?